contribs/gmf/src/sass/vars_only.scss

```diff
@@ -34,13 +34,18 @@ $above-all: 9999;
 
 $nav-width: 20rem !default;
 
-$border-radius-base: 2px !default;
-$border-radius-small: 1px !default;
+$border-radius-base: 0 !default;
 $input-border-focus: darken($brand-primary, $standard-variation) !default;
 
 $fa-font-path: "~font-awesome/fonts";
 
 // Bootstrap 3 compatibility layer
+$border-radius: $border-radius-base !default;
+$border-radius-lg: $border-radius-base !default;
+$border-radius-sm: $border-radius-base !default;
+$font-size-base: 0.8rem !default;
+$input-btn-padding-y: 0.25rem !default;
+$input-btn-padding-x: 0.5rem !default;
```

Comment:
Shouldn't we set the comment
`# Bootstrap 4 customization`
for this block?

Reply:
You mean you want that I change the `// Bootstrap 3 compatibility layer` comment to yours ?